src/App.tsx

```diff
@@ -1,54 +1,89 @@
-import React from 'react';
+import React, { useState, useMemo, useCallback } from 'react';
 import './App.scss';
 import { peopleFromServer } from './data/people';
+import { Person } from './types/Person';
+
+// eslint-disable-next-line @typescript-eslint/ban-types
+const debounce = (f: Function, delay: number) => {
+  let timerId: any;
```

Comment:
Try to fix all linter errors

Reply:
This is recommendation mentor

Reply:
you shouldn't disable eslint, so please delete type any and that